adding servicepulse and serviceinsight support policy (#5806)
Particular/docs.particular.net: serviceinsight/upgrades/support-policy.md

---

> title: Support Policy

Reviewer: We have too many docs with the very same title, makes it hard to find the one you want!
```suggestion
title: ServiceInsight support policy
```

summary: Versions of ServiceInsight that are currently supported
reviewed: 2022-02-18
related:
 - nservicebus/upgrades/release-policy
 - nservicebus/upgrades/supported-versions
 - nservicebus/upgrades/supported-platforms
 - servicecontrol/upgrades/supported-versions
---

> - Major versions are supported for a period of one year after the release of the next major version.

Reviewer: This is weird because we've never done a major of ServiceInsight. I'm wondering why we'd ever have a breaking change—it's a GUI app completely dependent upon an API that has its own support policy.

I'm actually going to challenge this - I think only the latest version should be supported, period. Until we have an actual reason to release a 2.0 at which point we could revisit the policy.

Reviewer 2: Updated the text to indicate only the latest version is supported.

- Only the latest minor version of a previous major version is supported.

> - Patch versions similarly go out of support when a new patch or minor versions are released.

Reviewer: ```suggestion
- Patch versions similarly go out of support when a new patch or minor version is released.
```

> For a list of supported ServiceInsight versions, refer to [supported versions](supported-versions.md).

Reviewer: This document is so short, why not put the supported versions on THIS page?

---
title: ServiceInsight support policy
summary: Versions of ServiceInsight that are currently supported
reviewed: 2022-08-05
related:
 - nservicebus/licensing
 - nservicebus/upgrades/all-versions
 - nservicebus/upgrades/support-policy
 - nservicebus/upgrades/release-policy
 - nservicebus/upgrades/supported-versions
 - nservicebus/upgrades/supported-platforms
 - servicecontrol/upgrades/support-policy
 - servicecontrol/upgrades/supported-versions
---

> Only the latest version of ServiceInsight is supported.

Reviewer: Thinking more, I don't think we need a grid like we do with downstreams and ServiceControl. As long as it's "Only the latest version" then we can just point to what that version is.

```suggestion
Only the latest version of ServiceInsight is supported. The latest version can be found on the [downloads page](https://particular.net/downloads).
```

Then there's also no include file to have to keep up-to-date. (Until we find a reason that we really do need that.)

> ## ServiceInsight supported versions
> 
> The following table lists both currently and recently supported versions of ServiceInsight.
> 
> include: supported-versions-serviceinsight

Reviewer: ```suggestion
```
